lsp-pwsh.el

```diff
@@ -257,6 +262,32 @@ Must not nil.")
 (defun lsp-pwsh--extra-init-params ()
   "Return form describing parameters for language server.")
 
+(defun lsp-pwsh--apply-code-action-edits (action)
+  "Handle ACTION for PowerShell.ApplyCodeActionEdits."
+  (-if-let* ((command (gethash "command" action))
+             ((&hash "StartLineNumber" "EndLineNumber"
+                     "StartColumnNumber" "EndColumnNumber" "Text")
+              (aref (gethash "arguments" action) 0))
+             (start (lsp--line-character-to-point (- StartLineNumber 1)
+                                                  (- StartColumnNumber 1)))
+             (end (lsp--line-character-to-point (- EndLineNumber 1)
+                                                (- EndColumnNumber 1))))
+      (save-excursion
```

Comment:
Just want to warn you that this technic of applying changes is causing issues with lenses and it is a bit less efficient compared to what lsp-mode is using (see lsp--apply-text-edit-replace-buffer-contents). I doubt that this will be a problem for powershell.

Reply:
I see, I've created an `text-edit` from arguments and pass it to `lsp--apply-text-edits`